docs/api/ref/DesktopAgent.md

```diff
@@ -383,8 +389,18 @@ if (fdc3.getInfo && versionIsAtLeast(await fdc3.getInfo(), "1.2")) {
 }
 ```
 
+The `ImplementationMetadata` object returned also includes the metadata for the calling application, according to the desktop agent. This allows the application to retrieve its own `appId`, `instanceId` and other details, e.g.:
+
+```js
+let implMetadata = await fdc3.getInfo();
+let {appId, instanceId} = impleMetadata.appMetadata;
```

Comment:
```suggestion
let implementationMetadata = await fdc3.getInfo();
let {appId, instanceId} = implementationMetadata.appMetadata;
```

Right now one has an 'e' in 'imple' and the other doesn't. Can also use the shorter names, I'm just a fan of verbose variable names.

Reply:
Me too generally, happy to fix